docs/developer-docs/latest/update-migration-guides/migration-guides/v4/migration-guide-4.3.x-to-4.4.x.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Migrate from 4.2.x to 4.3.x - Strapi Developer Docs
+description: Learn how you can migrate your Strapi application from 4.3.x to 4.4.x.
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/update-migration-guides/migration-guides/v4/migration-guide-4.3.x+-to-4.4.x.html
+---
+
+# v4.3.x to v4.4.x migration guide
+
+The Strapi v4.3.x to v4.4.x migration guide upgrades versions of v4.3.x and above to v4.4.x. This migration guide is needed for all users who were limiting media size for the local upload provider. The migration to 4.4.x consists of 3 steps:
+
+- Upgrading the application dependencies
+- Updating the local upload provider sizeLimit
```

Comment:
```suggestion
- Updating the local upload provider `sizeLimit`
```

I'll also mention/suggest it with every occurrence of the word. We should use inline code formatting as it's the actual name of a property (would be "size limit" in 'pure' English, but `sizeLimit` for the property name). 😊